tools_docker/Debian_Uwsgi/apache2.conf.sample

```diff
@@ -26,20 +26,32 @@
         Dav On
     </Directory>
     SetEnv proxy-sendcl
-    ProxyPreserveHost On
 
-    #ProxyPass /webdav http://127.0.0.1:3030/webdav
-    #ProxyPassReverse /webdav http://127.0.0.1:3030/webdav
+    # Setting Destination header from https to http in proxy
+    # is needed for working move/copy in webdav
+    RequestHeader edit Destination ^https http early
 
+    # Proxying Webdav
+    <Location /webdav>
+        # Preserving host is needed for working move/copy in webdav
+        ProxyPreserveHost On
+        #ProxyPass http://127.0.0.1:3030/webdav
```

Comment:
Not sure about this strategy to enable/disable webdav using sed. May be better to remove all location but more complicated.

Reply:
I have seen docker removing configuration on another project (jitsi meet) and its not a good idea for administration. You not understand what is available for the app.

If all parameter are visible but commented, you can understand what is possible to do. Maybe understand what is missing.

If we remove config if webdav is not started, if you activate webdav but is not working (because this line is not added in the file) you need to find what you need to do to use webdav with Tracim.

If config is visible, you can try to uncomment manually, or you can understand what is missing for Tracim.

Reply:
Another possibility would be to use additional config files for webdav/... and enable them through `a2enconf`. This has the advantage of being more robust than a sed based solution (as the script calling sed has to be updated whenever a change is made in the configuration it edits). But you end up with several configuration files to read (which is a matter of habits I think).

Reply:
I'm not sure is more simple (for basic user) to use multiple configuration files. Its better than use `sed` (like you explain) but not sure is more simple for support.

In the futur (we have discuss about that with @lebouquetin, but not validate choice), we probably won't continue to leave all configuration file (apache, uwsgi, development.ini) directly in docker volume. So when we do this modification we can improve usage of configuration files (like you purpose for apache).